scalingo/utils_slice.go

```diff
@@ -1,5 +1,18 @@
 package scalingo
 
+// Filter filters slice by using the test function.
+// It returns a filtered slice.
+func Filter[S any](slice []S, test func(S) bool) []S {
```

Comment:
I don't see a good reason to make this public, is there?
```suggestion
func filter[S any](slice []S, test func(S) bool) []S {
```